### ISSUE TITLE
telephone.wav does not import in production

### ISSUE THREAD

**Issue by azimpradhan** [ProjectCodaW/CoDaw#21]
https://github.com/ProjectCodaW/CoDaw/commit/cf63897b17e08b21caac4822331fc5a938733a56#diff-7f6ff1a223ded2b785756bcd68344a72R27

the latest deploy can be viewed at:
dev.codaw.co
Open the chrome console to see the error.

**Comment by felakuti4life** [comment]
Do we want it to be deployed on production?

Currently it is set up like so:

```
if(app.env === 'development') {
    app.import('vendor/telephone.wav', {
      destDir: 'assets'
    });
  }
```

**Comment by azimpradhan** [comment]
@felakuti4life, I think all features that we make in development should be able to be viewed in production as well. The idea is that we deploy often and make sure that features (such as sound) work as they should in all environments. This avoids having a gap between our work in our development environment and our production environment. In the code above, we should handle adding the asset url for production as well.

**Comment by mattahorton** [comment]
telephone.wav is dummy data. It's not exactly a feature.

**Comment by azimpradhan** [comment]
@mattahorton, telephone.wav was checked into stage, and according to our setup, the stage branch deploys to dev.codaw.co In order to keep our production and development environments in sync, we should write code that works in both environments. I agree with you that telephone.wav is dummy data. But it allows us to test your work on the tracks in development, and it should allow for the same in production.

**Comment by azimpradhan** [comment]
the latest deploy, which can be seen at dev.codaw.co fails because of the undefined route

**Comment by azimpradhan** [comment]
Also if there is any confusion about what I mean by production, it just means the environment in which codaw is deployed. On our local computers, the environment is development, but on dev.codaw.co (which tracks the branch stage) and codaw.co (which tracks the branch master), the environment is production. I think we want want telephone.wav to show up on dev.codaw.co to ensure that all our work is deployable. Of course we won't deploy it it to codaw.co until we remove it in a future commit, and replace it with actual sound. This is better than having code that says 

```
if environment === 'development' then do blah blah blah
```

because it is guaranteed to break when deployed

**Comment by mattahorton** [comment]
Fixed by #28
